components/hero/hero.sass

```diff
@@ -1,22 +1,17 @@
 .hero
   position: relative
-  height: 100vh
+  height: 85vh
   width: 100%
   display: flex
   align-items: center
   justify-content: center
 
-.hero__background
-  background-image: url('https://images.unsplash.com/photo-1550847067-93887e03cfbe?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=3000&q=80')
+.hero__logo
+  background-image: url('https://i.imgur.com/qrovqR0.png')
   background-size: cover
   background-repeat: no-repeat
   background-position: center
   position: absolute
-  height: 100%
-  width: 100%
-  z-index: -1
-
-.hero__header
-    font-size: 5rem
-    font-weight: 400
-    color: white
\ No newline at end of file
+  height: 40%
```

Comment:
Let's add a mixin for our SASS



=size-square($size)
  height: $size
  width: $size

Reply:
Will make it easier to update later on, saves us from manually synchronizing values.